tmt/checks/journal.py

```diff
@@ -388,10 +323,18 @@ def after_test(
         logger: tmt.log.Logger,
     ) -> list[CheckResult]:
         if not invocation.guest.facts.has_systemd:
-            return [CheckResult(name='journal', result=ResultOutcome.SKIP)]
+            return [
+                CheckResult(
+                    name='journal', result=ResultOutcome.SKIP, note=['systemd not available']
```

Comment:
Same as above, we have nicer notes, see similar one in the AVC check, https://github.com/teemtee/tmt/blob/main/tmt/checks/avc.py#L465

Reply:
resolved

Reply:
checking